abcclassroom/quickstart.py

```diff
@@ -78,28 +78,44 @@ def create_dir_struct(course_name="abc_course", force=False, working_dir=None):
                 Consider using a different course name, deleting the
                 existing directory, or running quikstart with the -f flag to
                 force overwrite the existing directory.""".format(
-                    course_name
+                    main_dir
                 )
             )
-    # make the main course directory and copy the config file there
+    # Make the main course directory and copy the config file there
     main_dir.mkdir()
     copy(config_path, main_dir)
 
-    # now we can use config functions to read / write config
+    # Use config functions to read / write config
+    # TODO: Can't we just copy the sample config - why are we rewriting it
+    #  here?
     config = cf.get_config(main_dir)
     config["course_directory"] = str(main_dir)
     cf.write_config(config, main_dir)
     clone_dir = cf.get_config_option(config, "clone_dir")
     template_dir = cf.get_config_option(config, "template_dir")
 
-    # make the required subdirectories
+    # Make the required subdirectories
     Path(main_dir, clone_dir).mkdir()
     Path(main_dir, template_dir).mkdir()
 
-    # create the extra_files directory in the main_dir
+    # Create the extra_files directory in the main_dir & copy files
     extra_path = path_to_example("extra_files")
     copytree(extra_path, Path(main_dir, "extra_files"))
 
+    # Copy the sample roster over to the new quickstart dir
```

Comment:
quickstart was NOT copying this file over before so i added this but it's not tested